tools/interop_matrix/run_interop_matrix_tests.py

```diff
@@ -163,7 +163,8 @@ def run_tests_for_lang(lang, runtime, images):
     # Download the docker image before running each test case.
     subprocess.check_call(['gcloud', 'docker', '--', 'pull', image])
     _docker_images_cleanup.append(image)
-    job_spec_list = find_test_cases(lang,release)
+    suite_name = '%s__%s_%s' % (lang, runtime, release)
```

Comment:
This got changed from `'%s__%s %s'` to `'%s__%s_%s'`. Is this intentional?

Reply:
Yes, it's to avoid any potential issues with ' ' in terms of rendering and searching.